src/v2/BaseLoginRouter.js

```diff
@@ -32,11 +32,21 @@ import AppState from './models/AppState';
 import idx from 'idx';
 
 
-const introspectStateToken = (settings) => {
+const introspect = (settings) => {
+  const clientId = settings.get('clientId');
   const domain = settings.get('baseUrl');
+  const scopes = settings.get('scopes');
   const stateHandle = settings.get('stateToken');
+  const redirectUri = settings.get('redirectUri');
   const version = settings.get('apiVersion');
-  return idx.start({ domain, stateHandle, version });
+  
+  if (stateHandle) {
```

Comment:
what if `stateHandle` and `useInteractionCodeFlow` both are present ? In that case we always call `introspect` ?

Reply:
I think Brett's comment can answer your question: https://github.com/okta/okta-signin-widget/pull/1533#discussion_r530637890

Reply:
If both are present, we should behave as if just stateHandle is present and ignore useInteractionCodeFlow.   This should only happen if there is a customized-okta-hosted widget, and in such a case we want to end with a redirect, so ignore useInteractionCodeFlow.

Reply:
@bretterer do we always call `introspect` with both exists ?